llama/generation.py

```diff
@@ -271,6 +271,10 @@ def generate(
 
         print(f"Processed prompts with {min_prompt_len} to {max_prompt_len} tokens, and generated {cur_pos_tensor.item() - max_prompt_len} tokens")
         print(f"Totally decoded {total_len - 1} tokens in {time.time() - decoding_start_time:.5f} seconds")
+        if ((total_len - 1)/({time.time() - decoding_start_time:.5f})) > 47.70 or ((total_len - 1)/({time.time() - decoding_start_time:.5f})) < 6.14:
```

Comment:
Can we use a variable for the metrics? And use some expressions for the upper and lower bounds?

Also, can you check with Jack or Will about how to make the metrics check on the XL-ML level instead of within the test itself?